### PR TITLE
HUGE PR: nextJS Rewrite

### PR DESCRIPTION
**Pull request by Looskie**
Practically fixed all your issues here in this pr with the website

its faster, better, stronger, new animations, its pretty, it will be the website that you've always dreamt of, all in all, you're welcome.

this isn't ready for merge, just setting this PR up

### CODE REVIEW

**Comment by pxseu** [comment]
LGTM!1

**Comment by cnrad** [comment]
This is actually really clean, insanely good work Looskie :+1:

**Comment by Looskie** [comment]
> This is actually really clean, insanely good work Looskie 👍

thanks cnraad1

**Comment by pxseu** [comment]
You're right cnraaaaaaaaaad

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
```
Comment (SincerelyFaust) [inline .gitignore]: I like this change

**Review by SincerelyFaust — APPROVED** [review]

**Comment by larsb24** [comment]
Yoo you legend!

**Comment by hexa-one** [comment]
Nice!

**Comment by SincerelyFaust** [comment]
Fixes #34

**Comment by Looskie** [comment]
b892c9b no clue how this actually is faster, but it is, sooo 💯

**Comment by Looskie** [comment]
Closing in favor for #44

**Pull request closed without merging**